lib/location/rest/rest_here_wifi.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file rest_here_wifi.h
+ *
+ * @brief Here REST API for WiFi positioning.
+ */
+
+#ifndef REST_HERE_WIFI_H
+#define REST_HERE_WIFI_H
+
+#include "rest_services_wifi.h"
+
+/**
+ * @brief Here WiFi positioning request.
+ *
+ * @param[in]     rcv_buf     Buffer for storing the REST service response.
+ * @param[in]     rcv_buf_len Length of the provided buffer.
+ * @param[in]     request     Data to be provided in API call.
+ * @param[in,out] result      Parsed results of API response.
+ *
+ * @retval 0 If successful.
+ *          Otherwise, a (negative) error code is returned.
+ */
+int here_rest_wifi_pos_get(
```

Comment:
We need to talk about the naming as filename is `rest_here_wifi` and functions are `here_rest_wifi`.

Reply:
I'll rename also functions based on new names in wifi dir.